core/gather/gatherers/full-page-screenshot.js

```diff
@@ -130,16 +130,16 @@ class FullPageScreenshot extends FRGatherer {
     });
     const data = 'data:image/webp;base64,' + result.data;
 
-    const documentSize =
-      await context.driver.executionContext.evaluate(getDocumentSize, {
+    const screenshotAreaSize =
+      await context.driver.executionContext.evaluate(getScreenshotAreaSize, {
```

Comment:
This should usually be the same as the document size. In some cases (e.g. infinite scroll) the document element could grow larger after we expand the page size.

Reply:
This sounds like a good comment to add.